test/registry/jurors-registry-staking.js

```diff
@@ -319,6 +350,21 @@ contract('JurorsRegistry', ([_, juror, juror2, jurorUniqueAddress, juror2UniqueA
         //     await assertRevert(registry.stake(MIN_ACTIVE_AMOUNT, data, { from }), 'JR_SENDER_NOT_VERIFIED')
```

Comment:
Should we remove all these commented tests?

Reply:
Nah not yet, I'll get to it.